UnitTests/ConsoleDrivers/ContentsTests.cs

```diff
@@ -17,40 +17,40 @@ public ContentsTests (ITestOutputHelper output)
 		this.output = output;
 	}
 
-	[Theory]
+	[Theory, AutoInitShutdown]
 	[InlineData (typeof (FakeDriver))]
 	//[InlineData (typeof (NetDriver))]
 	//[InlineData (typeof (CursesDriver))]
 	//[InlineData (typeof (WindowsDriver))]
 	public void AddStr_With_Combining_Characters (Type driverType)
 	{
-		var driver = (ConsoleDriver)Activator.CreateInstance (driverType);
-		Application.Init (driver);
-		// driver.Init (null);
+		Application.Init ();
```

Comment:
This should not be needed for ConsoleDriver tests.

Reply:
Yes, it's needed because it was getting failure on another unit test due any racing condition while this unit test is running. Running the other unit test alone it doesn't give any failure but running all the unit tests I get failing on the other unit test. I don't remember the name of the other fail unit test. But try to run without `Application.Init` and `AutoInitShutdown` and check for yourself.

Reply:
That's a bug in the unit test system then. Let's figure that out instead of coupling Application to ConsoleDriver.

Reply:
You have here a dependence on `Application`, but I'll try to fix this.
`Application.Init (driver);`

Reply:
See my PR I just submitted to this PR.